back-end/routes/users.js

```diff
@@ -1,70 +1,70 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
 
-const { allUsers, checkUserLogin, checkUserQuests, getAllUserClassProgress, getUser, getBadgesByUser, allVillagers, allAdventurers } = require('../db/helpers');
-
-
+const {
+  allUsers,
+  checkUserLogin,
+  checkUserQuests,
+  getAllUserClassProgress,
+  getUser,
+  getBadgesByUser,
+  allVillagers,
+  allAdventurers,
+} = require("../db/helpers");
 
 module.exports = () => {
-
   router.get("/users", (req, res) => {
-    allUsers().then(result => {
+    allUsers().then((result) => {
```

Comment:
I wonder: why is the parenthesis a rule here? Isn't it superfluous in this case since there is only one argument?

Reply:
https://github.com/prettier/prettier/issues/812

Happy reading :)

Reply:
Fair enough.